src/main/java/seedu/address/logic/commands/SummaryCommand.java

```diff
@@ -26,6 +26,8 @@ public class SummaryCommand extends Command {
             + "Example: " + COMMAND_WORD + " " + PREFIX_START_MONTH + "2024-09 " + PREFIX_END_MONTH + "2024-12";
 
     public static final String MESSAGE_SUCCESS = "The total amount of transactions from %s to %s is: %s";
+    public static final String MESSAGE_NO_TRANSACTIONS_FOUND = "No transactions found from %s to %s";
+
```

Comment:
I think we're not allowed to change or add any messages shown to the user because of the feature freeze, except the particular cases shown here.
![image](https://github.com/user-attachments/assets/b6eba870-e293-42ed-8fba-239dd10b1144)

Reply:
Hmmm ok so should I just close the issue? Or maybe it qualifies as fails silently? 😅

Reply:
I think its not allowed idk. I think what they meant by "fail silently" is like an error occured but no message to the user on what happened

Reply:
Ok removed the changes in [e0b67e4](https://github.com/AY2425S1-CS2103T-F13-4/tp/pull/257/commits/e0b67e4900a185b7b3169018799f0bf31f73ed20)